app/build.gradle.kts

```diff
@@ -204,7 +204,7 @@ dependencies {
     implementation("com.mikepenz:iconics-compose:5.4.0")
     implementation("com.mikepenz:community-material-typeface:7.0.96.0-kotlin@aar")
 
-    "fullImplementation"("org.burnoutcrew.composereorderable:reorderable:0.9.6")
+    implementation("org.burnoutcrew.composereorderable:reorderable:0.9.6")
```

Comment:
This should be added to Automotive as well

Reply:
looks like it already is :)

https://github.com/home-assistant/android/blob/2804e7c4a854c64792196374bb728b01d082900e/automotive/build.gradle.kts#L236